src/t_mockers.rs

```diff
@@ -140,11 +140,19 @@ impl TestSuite for Mockers {
     }
 
     fn foreign() {
-        unimplemented!()
```

Comment:
mock_derive can also mock external Rust functions, not just C functions.  Can Mockers do that too?

Reply:
Do you mean `extern "Rust" { … }`? It works.

Reply:
Could you please add that to the test, so it's functionally equivalent to `t_mock_derive::t::foreign`?